src/pk/dh/dh_import.c

```diff
@@ -72,6 +74,10 @@ int dh_import(const unsigned char *in, unsigned long inlen, dh_key *key)
          mp_clear(key->x);
```

Comment:
Although unrelated to the main idea of this PR I think that it would be better to:
```
mp_set_int(key->x, 0);
```
instead of:
```
mp_clear(key->x);
key->x = NULL;
```

Reply:
uhm, why?

Reply:
It might got a bit tricky as we use `*_multi` functions where we use NULL as an end of args marker. Like in `dh_free`:
```C
mp_cleanup_multi(&key->prime, &key->base, &key->y, &key->x, NULL);
```
which means that setting `key->x` to NULL is fine but only till somebody will add an extra stuff like `..., &key->x, &key->extra, NULL);` (which can easily happen as there is no comment in `dh_free` that `key->x` must be the last arg of `mp_cleanup_multi`).

We do not use this approach anywhere else and I am not sure if it is so useful.

Reply:
makes sense

Reply:
I decided to apply the change directly on the release branch, but didn't merge this PR as I'm doubting its usefulness...